Formula/blast.rb

```diff
@@ -12,11 +12,27 @@ class Blast < Formula
     sha256 "57f2e2f9c65aa5364eb72a9bbdf6948a30af2975a53445a4b43203f56395da7c" => :sierra
   end
 
+  depends_on "edirect"
```

Comment:
This formula does not exist in homebrew-core.

Reply:
Indeed, thanks for pointing this out. This is available at the link below. How can I specify such a dependency?
https://github.com/brewsci/homebrew-bio/blob/master/Formula/edirect.rb

Reply:
If the dependency is indeed needed, either `edirect` should be available in homebrew-core or this formula should be removed (effectively migrating it to homebrew-bio).

If it was a trivial dependency then pulling in a resource and working with that could be an option, but I don't think `edirect` falls under that category.

Reply:
Thanks. How do you recommend we proceed?